boards/arm/mimxrt1050_evk/doc/mimxrt1050_evk.rst

```diff
@@ -0,0 +1,176 @@
+.. _mimxrt1050_evk:
+
+NXP MIMXRT1050-EVK
+##################
+
+Overview
+********
+
+The i.MX RT1050 is a new processor family featuring NXP's advanced
+implementation of the ARM Cortex-M7 Core. It provides high CPU performance and
+best real-time response. The i.MX RT1050 provides various memory interfaces,
+including SDRAM, Raw NAND FLASH, NOR FLASH, SD/eMMC, Quad SPI, HyperBus and a
+wide range of other interfaces for connecting peripherals, such as WLAN,
+Bluetooth™, GPS, displays, and camera sensors. Same as other i.MX processors,
+i.MX RT1050 also has rich audio and video features, including LCD display,
+basic 2D graphics, camera interface, SPDIF and I2S audio interface.
+
+.. image:: mimxrt1050_evk.jpg
+   :width: 720px
+   :align: center
+   :alt: MIMXRT1050-EVK
+
+Hardware
+********
+
+- MIMXRT1052DVL6A MCU (600 MHz, 512 KB TCM)
+- Memory
+  - 256 KB SDRAM
```

Comment:
Need a blank like before a sub-list or it's picked up as a continuation of the previous line.